app/forms/ctc/phone_verification_form.rb

```diff
@@ -5,6 +5,8 @@ class PhoneVerificationForm < QuestionsForm
     validates_presence_of :verification_code
 
     def valid?
+      return true if verification_code = "000000" && (Rails.env.demo? || Rails.env.development?)
```

Comment:
I think this should be `==`. I think that the way it's written (`=` vs. `==`), it will accept any code on demo/dev.